spec/rails/doc/smart/openapi.yaml

```diff
@@ -164,6 +180,8 @@ paths:
                   storage_size:
                     type: number
                     format: float
+                  no_such_field_response:
+                    type: boolean
```

Comment:
Note) This is added but does not appear in the`expected.json` because it is removed because of the pattern `7) paths.*.*.responses.*.content.application/json.schema.properties.*`